src/h5web/visualizations/nexus/utils.ts

```diff
@@ -30,13 +31,52 @@ export function getAttributeValue(
   return entity.attributes.find((attr) => attr.name === attributeName)?.value;
 }
 
+export function getChildEntityByName(
+  group: HDF5Group,
+  metadata: HDF5Metadata,
+  entityName: string | undefined
```

Comment:
This should probably be `string` -- just make sure you don't call it with an undefined `entityName`.